Update 01-introduction.md (#84)
SoftDev4Research/4OSS-lesson: _episodes/01-introduction.md

"_When all researchers are aware of Open Science, and are trained, supported and guided at all career stages to practice Open Science, the potential is there to fundamentally change the way research is performed and disseminated. Fostering a scientific ecosystem in which research gains increased visibility, is shared more efficiently, and is performed with enhanced research integrity._" [Open Science Skills Working Group Report (2017)](https://ec.europa.eu/research/openscience/pdf/os_skills_wgreport_final.pdf#view=fit&pagemode=none)

> Discussing best practices to develop research software is bound to touch on the subject of Open Science. Modern research relies on software, and building upon or reproducing that research requires access to the full source code behind that software ([ref](https://open-science-training-handbook.gitbook.io/book/examples-and-practical-guidance)). Sharing software used for research is a necessary, though not sufficient, condition for reproducibility. In addition to reproducibility, sharing software openly allows developers/scientists to receive career credit for their efforts, either through direct citation or via published software articles. Hence, why it is important to have a license and citation, you are going to be discussing all of these aspects in the following lesson.

Reviewer: ```suggestion
Discussing best practices to develop research software is bound to touch on the subject of Open Science. Modern research relies on software, and building upon or reproducing that research requires access to the full source code behind that software ([ref](https://open-science-training-handbook.gitbook.io/book/examples-and-practical-guidance)). Sharing software used for research is one of the requirements for reproducibility. In addition to reproducibility, sharing software openly allows developers/scientists to receive career credit for their efforts, either through direct citation or via published software articles.
```
